diff -ruN kodi-pvr-hts-9999.orig/CMakeLists.txt kodi-pvr-hts-9999/CMakeLists.txt
--- kodi-pvr-hts-9999.orig/CMakeLists.txt	2015-12-21 01:51:14.130578030 +0100
+++ kodi-pvr-hts-9999/CMakeLists.txt	2015-12-21 01:51:31.816799154 +0100
@@ -6,7 +6,7 @@
 
 find_package(kodi REQUIRED)
 find_package(kodiplatform REQUIRED)
-find_package(platform REQUIRED)
+find_package(p8-platform REQUIRED)
 
 include_directories(${kodiplatform_INCLUDE_DIRS}
                     ${platform_INCLUDE_DIRS}
diff -ruN kodi-pvr-hts-9999.orig/src/AsyncState.cpp kodi-pvr-hts-9999/src/AsyncState.cpp
--- kodi-pvr-hts-9999.orig/src/AsyncState.cpp	2015-12-21 01:51:14.137578117 +0100
+++ kodi-pvr-hts-9999/src/AsyncState.cpp	2015-12-21 01:54:53.407323288 +0100
@@ -26,7 +26,7 @@
   AsyncState *self;
 };
 
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 
 AsyncState::AsyncState(int timeout)
 {
diff -ruN kodi-pvr-hts-9999.orig/src/AsyncState.h kodi-pvr-hts-9999/src/AsyncState.h
--- kodi-pvr-hts-9999.orig/src/AsyncState.h	2015-12-21 01:51:14.137578117 +0100
+++ kodi-pvr-hts-9999/src/AsyncState.h	2015-12-21 01:53:50.206531191 +0100
@@ -22,7 +22,7 @@
 #ifndef ASYNCSTATE_H
 #define	ASYNCSTATE_H
 
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/mutex.h>
 
 /**
  * Represents the possible states
@@ -53,7 +53,7 @@
    */
   inline eAsyncState GetState()
   {
-    PLATFORM::CLockObject lock(m_mutex);
+    P8PLATFORM::CLockObject lock(m_mutex);
     return m_state;
   }
 
@@ -76,8 +76,8 @@
   static bool PredicateCallback ( void *param );
   
   eAsyncState m_state;
-  PLATFORM::CMutex m_mutex;
-  PLATFORM::CCondition<bool> m_condition;
+  P8PLATFORM::CMutex m_mutex;
+  P8PLATFORM::CCondition<bool> m_condition;
   int m_timeout;
 
 };
diff -ruN kodi-pvr-hts-9999.orig/src/AutoRecordings.cpp kodi-pvr-hts-9999/src/AutoRecordings.cpp
--- kodi-pvr-hts-9999.orig/src/AutoRecordings.cpp	2015-12-21 01:51:14.137578117 +0100
+++ kodi-pvr-hts-9999/src/AutoRecordings.cpp	2015-12-21 01:54:44.111206737 +0100
@@ -25,7 +25,7 @@
 #include "tvheadend/Settings.h"
 #include "tvheadend/utilities/Utilities.h"
 
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 using namespace tvheadend;
 using namespace tvheadend::entity;
 
diff -ruN kodi-pvr-hts-9999.orig/src/client.cpp kodi-pvr-hts-9999/src/client.cpp
--- kodi-pvr-hts-9999.orig/src/client.cpp	2015-12-21 01:51:14.139578142 +0100
+++ kodi-pvr-hts-9999/src/client.cpp	2015-12-21 01:53:33.686324259 +0100
@@ -22,13 +22,13 @@
 #include "client.h"
 #include "kodi/xbmc_pvr_dll.h"
 #include "kodi/libKODI_guilib.h"
-#include "platform/util/util.h"
+#include <p8-platform/util/util.h>
 #include "Tvheadend.h"
 #include "tvheadend/Settings.h"
 
 using namespace std;
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 using namespace tvheadend;
 
 /* **************************************************************************
diff -ruN kodi-pvr-hts-9999.orig/src/client.h kodi-pvr-hts-9999/src/client.h
--- kodi-pvr-hts-9999.orig/src/client.h	2015-12-21 01:51:14.139578142 +0100
+++ kodi-pvr-hts-9999/src/client.h	2015-12-21 01:51:31.817799167 +0100
@@ -20,8 +20,8 @@
  *
  */
 
-#include "platform/os.h"
-#include "platform/threads/mutex.h"
+#include <p8-platform/os.h>
+#include <p8-platform/threads/mutex.h>
 #include "kodi/libXBMC_addon.h"
 #include "kodi/libXBMC_pvr.h"
 #include "kodi/libXBMC_codec.h"
diff -ruN kodi-pvr-hts-9999.orig/src/HTSPConnection.cpp kodi-pvr-hts-9999/src/HTSPConnection.cpp
--- kodi-pvr-hts-9999.orig/src/HTSPConnection.cpp	2015-12-21 01:51:14.137578117 +0100
+++ kodi-pvr-hts-9999/src/HTSPConnection.cpp	2015-12-21 01:53:24.253206117 +0100
@@ -19,9 +19,9 @@
  *
  */
 
-#include "platform/threads/mutex.h"
-#include "platform/util/StringUtils.h"
-#include "platform/sockets/tcp.h"
+#include <p8-platform/threads/mutex.h>
+#include <p8-platform/util/StringUtils.h>
+#include <p8-platform/sockets/tcp.h>
 
 extern "C" {
 #include "libhts/htsmsg_binary.h"
@@ -32,7 +32,7 @@
 
 using namespace std;
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 using namespace tvheadend;
 
 /*
diff -ruN kodi-pvr-hts-9999.orig/src/HTSPDemuxer.cpp kodi-pvr-hts-9999/src/HTSPDemuxer.cpp
--- kodi-pvr-hts-9999.orig/src/HTSPDemuxer.cpp	2015-12-21 01:51:14.137578117 +0100
+++ kodi-pvr-hts-9999/src/HTSPDemuxer.cpp	2015-12-21 01:54:31.943054203 +0100
@@ -26,7 +26,7 @@
 
 using namespace std;
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 using namespace tvheadend;
 
 CHTSPDemuxer::CHTSPDemuxer ( CHTSPConnection &conn )
diff -ruN kodi-pvr-hts-9999.orig/src/HTSPVFS.cpp kodi-pvr-hts-9999/src/HTSPVFS.cpp
--- kodi-pvr-hts-9999.orig/src/HTSPVFS.cpp	2015-12-21 01:51:14.138578130 +0100
+++ kodi-pvr-hts-9999/src/HTSPVFS.cpp	2015-12-21 01:53:15.501096514 +0100
@@ -19,8 +19,8 @@
  *
  */
 
-#include "platform/threads/mutex.h"
-#include "platform/util/StringUtils.h"
+#include <p8-platform/threads/mutex.h>
+#include <p8-platform/util/StringUtils.h>
 
 extern "C" {
 #include "libhts/htsmsg_binary.h"
@@ -29,7 +29,7 @@
 #include "Tvheadend.h"
 
 using namespace std;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 
 /*
 * VFS handler
diff -ruN kodi-pvr-hts-9999.orig/src/TimeRecordings.cpp kodi-pvr-hts-9999/src/TimeRecordings.cpp
--- kodi-pvr-hts-9999.orig/src/TimeRecordings.cpp	2015-12-21 01:51:14.138578130 +0100
+++ kodi-pvr-hts-9999/src/TimeRecordings.cpp	2015-12-21 01:54:19.277895468 +0100
@@ -24,7 +24,7 @@
 #include "Tvheadend.h"
 #include "tvheadend/utilities/Utilities.h"
 
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 using namespace tvheadend;
 using namespace tvheadend::entity;
 
diff -ruN kodi-pvr-hts-9999.orig/src/tvheadend/Subscription.cpp kodi-pvr-hts-9999/src/tvheadend/Subscription.cpp
--- kodi-pvr-hts-9999.orig/src/tvheadend/Subscription.cpp	2015-12-21 01:51:14.139578142 +0100
+++ kodi-pvr-hts-9999/src/tvheadend/Subscription.cpp	2015-12-21 01:54:08.182756415 +0100
@@ -22,7 +22,7 @@
 #include "Subscription.h"
 #include "../Tvheadend.h"
 
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 using namespace tvheadend;
 
 Subscription::Subscription(CHTSPConnection &conn) :
diff -ruN kodi-pvr-hts-9999.orig/src/tvheadend/Subscription.h kodi-pvr-hts-9999/src/tvheadend/Subscription.h
--- kodi-pvr-hts-9999.orig/src/tvheadend/Subscription.h	2015-12-21 01:51:14.139578142 +0100
+++ kodi-pvr-hts-9999/src/tvheadend/Subscription.h	2015-12-21 01:54:01.325670496 +0100
@@ -22,7 +22,7 @@
  */
 
 #include <string>
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/mutex.h>
 
 extern "C"
 {
@@ -146,6 +146,6 @@
     std::string       m_profile;
     CHTSPConnection   &m_conn;
 
-    mutable PLATFORM::CMutex  m_mutex;
+    mutable P8PLATFORM::CMutex  m_mutex;
   };
 }
diff -ruN kodi-pvr-hts-9999.orig/src/Tvheadend.cpp kodi-pvr-hts-9999/src/Tvheadend.cpp
--- kodi-pvr-hts-9999.orig/src/Tvheadend.cpp	2015-12-21 01:51:14.138578130 +0100
+++ kodi-pvr-hts-9999/src/Tvheadend.cpp	2015-12-21 01:53:01.629922839 +0100
@@ -23,14 +23,14 @@
 #include <ctime>
 #include <memory>
 
-#include "platform/util/StringUtils.h"
+#include <p8-platform/util/StringUtils.h>
 
 #include "Tvheadend.h"
 #include "tvheadend/utilities/Utilities.h"
 
 using namespace std;
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 using namespace tvheadend;
 using namespace tvheadend::entity;
 
diff -ruN kodi-pvr-hts-9999.orig/src/Tvheadend.h kodi-pvr-hts-9999/src/Tvheadend.h
--- kodi-pvr-hts-9999.orig/src/Tvheadend.h	2015-12-21 01:51:14.138578130 +0100
+++ kodi-pvr-hts-9999/src/Tvheadend.h	2015-12-21 01:52:50.699786016 +0100
@@ -22,10 +22,10 @@
  */
 
 #include "client.h"
-#include "platform/sockets/tcp.h"
-#include "platform/threads/threads.h"
-#include "platform/threads/mutex.h"
-#include "platform/util/buffer.h"
+#include <p8-platform/sockets/tcp.h>
+#include <p8-platform/threads/threads.h>
+#include <p8-platform/threads/mutex.h>
+#include <p8-platform/util/buffer.h>
 #include "kodi/xbmc_codec_types.h"
 #include "kodi/xbmc_stream_utils.hpp"
 #include "kodi/libXBMC_addon.h"
@@ -106,7 +106,7 @@
 
 /* Typedefs */
 typedef std::map<uint32_t,CHTSPResponse*> CHTSPResponseList;
-typedef PLATFORM::SyncedBuffer<CHTSPMessage> CHTSPMessageQueue;
+typedef P8PLATFORM::SyncedBuffer<CHTSPMessage> CHTSPMessageQueue;
 
 /*
  * HTSP Response handler
@@ -116,10 +116,10 @@
 public:
   CHTSPResponse();
   ~CHTSPResponse();
-  htsmsg_t *Get ( PLATFORM::CMutex &mutex, uint32_t timeout );
+  htsmsg_t *Get ( P8PLATFORM::CMutex &mutex, uint32_t timeout );
   void      Set ( htsmsg_t *m );
 private:
-  PLATFORM::CCondition<volatile bool> m_cond;
+  P8PLATFORM::CCondition<volatile bool> m_cond;
   bool                                m_flag;
   htsmsg_t                           *m_msg;
 };
@@ -164,7 +164,7 @@
  * HTSP Connection registration thread
  */
 class CHTSPRegister
-  : public PLATFORM::CThread
+  : public P8PLATFORM::CThread
 {
   friend class CHTSPConnection;
 
@@ -181,7 +181,7 @@
  * HTSP Connection
  */
 class CHTSPConnection
-  : public PLATFORM::CThread
+  : public P8PLATFORM::CThread
 {
   friend class CHTSPRegister;
 
@@ -208,7 +208,7 @@
   inline bool IsConnected       ( void ) const { return m_ready; }
   bool        WaitForConnection ( void );
 
-  inline PLATFORM::CMutex& Mutex ( void ) { return m_mutex; }
+  inline P8PLATFORM::CMutex& Mutex ( void ) { return m_mutex; }
 
   void        OnSleep ( void );
   void        OnWake  ( void );
@@ -220,10 +220,10 @@
   bool        SendHello        ( void );
   bool        SendAuth         ( const std::string &u, const std::string &p );
 
-  PLATFORM::CTcpSocket               *m_socket;
-  PLATFORM::CMutex                    m_mutex;
+  P8PLATFORM::CTcpSocket               *m_socket;
+  P8PLATFORM::CMutex                    m_mutex;
   CHTSPRegister                       m_regThread;
-  PLATFORM::CCondition<volatile bool> m_regCond;
+  P8PLATFORM::CCondition<volatile bool> m_regCond;
   bool                                m_ready;
   uint32_t                            m_seq;
   std::string                         m_serverName;
@@ -291,13 +291,13 @@
   void SetStreamingProfile(const std::string &profile);
 
 private:
-  PLATFORM::CMutex                        m_mutex;
+  P8PLATFORM::CMutex                        m_mutex;
   CHTSPConnection                        &m_conn;
-  PLATFORM::SyncedBuffer<DemuxPacket*>    m_pktBuffer;
+  P8PLATFORM::SyncedBuffer<DemuxPacket*>    m_pktBuffer;
   ADDON::XbmcStreamProperties             m_streams;
   std::map<int,int>                       m_streamStat;
   int64_t                                 m_seekTime;
-  PLATFORM::CCondition<volatile int64_t>  m_seekCond;
+  P8PLATFORM::CCondition<volatile int64_t>  m_seekCond;
   tvheadend::status::SourceInfo           m_sourceInfo;
   tvheadend::status::Quality              m_signalInfo;
   tvheadend::status::TimeshiftStatus      m_timeshiftStatus;
@@ -366,7 +366,7 @@
  * Root object for Tvheadend connection
  */
 class CTvheadend
-  : public PLATFORM::CThread
+  : public P8PLATFORM::CThread
 {
 public:
   CTvheadend();
@@ -438,7 +438,7 @@
    */
   tvheadend::Profiles         m_profiles;
 
-  PLATFORM::CMutex            m_mutex;
+  P8PLATFORM::CMutex            m_mutex;
 
   CHTSPConnection             m_conn;
 
@@ -535,7 +535,7 @@
    */
   bool WaitForConnection ( void )
   {
-    PLATFORM::CLockObject lock(m_conn.Mutex());
+    P8PLATFORM::CLockObject lock(m_conn.Mutex());
     return m_conn.WaitForConnection();
   }
   std::string GetServerName    ( void )
