diff -ruN kodiplatform-9999.orig/CMakeLists.txt kodiplatform-9999/CMakeLists.txt
--- kodiplatform-9999.orig/CMakeLists.txt	2015-12-20 19:24:17.355760357 +0100
+++ kodiplatform-9999/CMakeLists.txt	2015-12-20 19:24:38.763176314 +0100
@@ -8,7 +8,7 @@
 find_package(kodi REQUIRED)
 find_package(TinyXML REQUIRED)
 find_package(Threads REQUIRED)
-find_package(platform REQUIRED)
+find_package(p8-platform REQUIRED)
 include(UseMultiArch.cmake)
 include(CheckAtomic.cmake)
 
diff -ruN kodiplatform-9999.orig/src/util/XMLUtils.h kodiplatform-9999/src/util/XMLUtils.h
--- kodiplatform-9999.orig/src/util/XMLUtils.h	2015-12-20 19:24:17.356760377 +0100
+++ kodiplatform-9999/src/util/XMLUtils.h	2015-12-20 19:25:36.021224624 +0100
@@ -21,7 +21,7 @@
  *
  */
 
-#include <platform/util/StdString.h>
+#include <p8-platform/util/StdString.h>
 #include "tinyxml.h"
 
 class XMLUtils
